crates/abq_test_support/abq_test_utils/src/lib.rs

```diff
@@ -104,14 +104,20 @@ pub fn wid(id: usize) -> WorkId {
     WorkId([id as u8; 16])
 }
 
+pub fn gid(id: usize) -> GroupId {
+    GroupId([id as u8; 16])
+}
+
 pub fn test(id: usize) -> TestId {
     format!("test{id}")
 }
 
+// only used in tests
```

Comment:
Not sure this comment is worth keeping around given that this is in a crate named `abq_test_support`.